client/src/components/span-table/spanData.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { useSelector } from 'react-redux';
+import { useAppDispatch } from '../../lib/hooks';
+import { selectSpanDataList } from './spanDataSlice';
+import '../../styles/spanTable.scss'
+
+const spanData= (props: any) => {
+ 
+  console.log(props) 
+  console.log('props.spanData.duration', props.spanData.duration)
```

Comment:
Leftover console logs? Or are these meant for future developers?

Reply:
removed